### DISCUSSION TITLE
Is that possible to disable Pydantic Schema generation?

### DISCUSSION THREAD

**Discussion by AlexanderFarkas** [pydantic/pydantic#8253] [Question] [answered]
I don't need Open API schemas in production. But pydantic still generates them on import. With limited memory and cpu it takes around 30 seconds.

**Comment by spyoungtech** [comment, marked as answer]
I'm not sure if this is exactly what you're asking, but I believe you can use the [`defer_build` option](https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.defer_build) for this.

```py
class MyModel(pydantic.BaseModel):
    foo: str = pydantic.Field(...)
    model_config = pydantic.ConfigDict(defer_build=True)
```

If you have a lot of models, especially nested models or forward references, this can significantly improve import time performance.
